### ISSUE TITLE
When starting, option to select to open one of the 5 recently open wallet without open file dialog. (when 1st one is password protected)

### ISSUE THREAD

**Issue by Transisto** [spesmilo/electrum#6661]
![image](https://user-images.githubusercontent.com/388716/96351699-3162f700-108b-11eb-9fe5-d8ccb9a7120d.png)

Just because the wallet that's opening has a password I can't choose which wallet we want to actually access without selecting the file in dialog or entering the password of that wallet.

**Comment by Transisto** [comment]
I don't think it make sense to have an **"Install Wizard"** windows when doing nothing else than opening the wallet just because it is password protected.

**Comment by SomberNight** [comment]
note: there was a PR like this at one point - https://github.com/spesmilo/electrum/pull/4069

**Comment by Transisto** [comment]
> note: there was a PR like this at one point - #4069

Very good implementation, wonder what happened to it.
